Cyral/Cyral.json

```diff
@@ -0,0 +1,1889 @@
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Auto",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel00FE41DCA5C91B44",
+                    "title": "Number of Users",
+                    "visualSettings": "{\"general\":{\"mode\":\"singleValueMetrics\",\"type\":\"svp\"},\"series\":{},\"svp\":{\"label\":\"Users\",\"noDataString\":\"0\"}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc )\n| json \"identity.group\",\"request.statementType\",\"activityTypes\",\"identity.endUser\" nodrop\n| where %\"identity.group\" matches \"{{GroupName}}\" AND %\"request.statementType\" matches \"{{StatementType}}\" AND %\"activityTypes\" CONTAINS \"query\"\n| count_distinct(%\"identity.endUser\")",
+                            "queryType": "Logs",
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Auto",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel3DE20BE5A4FF984B",
+                    "title": "Inactive Users",
+                    "visualSettings": "{\"general\":{\"mode\":\"singleValueMetrics\",\"type\":\"svp\"},\"series\":{},\"svp\":{\"label\":\"Users\",\"noDataString\":\"0\"}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc )\n| json \"identity.group\",\"request.statementType\",\"activityTime\",\"identity.endUser\",\"identity.repoUser\" nodrop\n| where %\"identity.group\" matches \"{{GroupName}}\" AND %\"request.statementType\" matches \"{{StatementType}}\"\n| first(%\"activityTime\") by %\"identity.endUser\", %\"identity.group\", %\"identity.repoUser\"\n| now()-30d as expireTime|parseDate(_first, \"yyyy-MM-dd HH:mm:ss\") as parsedDate\n| where expireTime > parsedDate\n| count",
```

Comment:
parseDate(_first, "yyyy-MM-dd'T'HH:mm:ss") as parsedDate
Here parseDate is giving warning  for this format yyyy-MM-dd HH:mm:ss

Reply:
I found sometimes activityTime has different format. 
![image](https://user-images.githubusercontent.com/3620468/127677037-bed918ec-55a5-4eb0-abe2-0f97ebe671af.png)

Reply:
I will fix my fabricated logs to show the correct date but I believe that these parseDate errors can be ignored  since they are erroring on my fabricated logs that have incorrect date formats.